types/intset_test.go

```diff
@@ -0,0 +1,90 @@
+package types_test
+
+import (
+	"math/rand"
+	"sort"
+	"testing"
+
+	tf "github.com/filecoin-project/go-filecoin/testhelpers/testflags"
+	"github.com/filecoin-project/go-filecoin/types"
+	"github.com/stretchr/testify/assert"
+)
+
+func TestIntSet(t *testing.T) {
+	tf.UnitTest(t)
+
+	t.Run("Add and Contains", func(t *testing.T) {
+		is := types.NewIntSet()
+		is.Add(1)
+		is.Add(0xffffffffffff)
+
+		assert.True(t, is.Contains(1))
+		assert.True(t, is.Contains(0xffffffffffff))
+		assert.False(t, is.Contains(2))
+	})
+
+	t.Run("IsSubset", func(t *testing.T) {
+		is0 := types.NewIntSet(1, 2, 3)
+		is1 := types.NewIntSet()
+
+		// {} ⊆ {1, 2, 3}
+		assert.True(t, is0.IsSubset(is1))
```

Comment:
The phrasing given by this method name is confusing, suggesting the converse relationship "is subset of". It would be less ambiguous as `is0.HasSubset(is1) == true`

Reply:
Oh yeah, that reads much better.